src/renderer/components/Meta.tsx

```diff
@@ -170,6 +172,18 @@ export default class Meta extends React.Component {
     });
   }
 
+  onChangeZoomLevel(scene: Scene, level: number) {
+    this.editScene(scene, (s) => {
+      s.zoomLevel = level;
+    });
+  }
+
+  onChangeHastebinID(scene: Scene, hbId: string) {
+    this.editScene(scene, (s) => {
+      s.hastebinID = hbId;
+    });
+  }
```

Comment:
I'm starting to think we should have `SceneDetail` just edit scenes directly and have one big `onChangeScene` prop that replaces the whole scene in the scenes list. Then we wouldn't have to add a new method for every single property update. (This is fine though.)

A framework like Redux would make the components smaller, but I really want to avoid Redux specifically because it's hard for a lot of people to understand, and I want to keep this code really accessible to newcomers.